### PR TITLE
Click event

### LINKED ISSUE #56: Clicking an Event
### Description
Right now, we have a static list of Events on the Events page. We want to be able to click on each one so we can see that Event's details and log hours for that specific Event.

this _may_ involve:  
 - using a `<Link />` for each of the `EventCard`s
 - passing the loaded Events from `App.tsx` into the `events.tsx` page
 - getting an event by Id when you navigate to a specific event page
 
### Checklist
- [ ] hovering over an Event makes the background color change (see the [figma](https://www.figma.com/file/2Ig3z2d3os0iniZ881RhHS/The-Land-Conservancy?node-id=312%3A2921))
- [ ] clicking on an event navigates to the corresponding `LogHours` page
- [ ] The `LogHours` page is populated with the correct event details

### PR DESCRIPTION
**Pull request by bbmcmann**
Clicking an event on the events page now navigates to the correct log hours page by event. Hovering over the event changes the background color as well. It does not work with the navbar component active, the pushed code has the navbar disabled on the events page. If there are any changes needed or other bug fixes I will work on those right away!
closes #56

### CODE REVIEW

File: frontend/src/components/pages/events.tsx [hunk 1]
```diff
@@ -11,41 +12,15 @@ const StyledContainer = styled(Container)`
   padding: 20px;
   align-items: left;
   justify-content: left;
+  text-decoration: none;
+  z-index: 100;
```
Comment (marissadarnell) [inline frontend/src/components/pages/events.tsx]: revert z-index changes

File: frontend/src/components/navigation/navComponents.tsx [hunk 2]
```diff
@@ -95,6 +95,10 @@ const LogoImage = styled.img`
   height: 113px;
 `;
 
+const StyledWrapper = styled.div`
+  z-index: 1;
+`;
+
```
Comment (marissadarnell) [inline frontend/src/components/navigation/navComponents.tsx]: can probably remove this

File: frontend/src/components/pages/eventCard.tsx [hunk 3]
```diff
@@ -39,6 +39,15 @@ const LayoutDiv = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
+  text-decoration: none;
+`;
+
+const StyledDiv = styled.div`
+  text-decoration: none;
+  z-index: 99999999;
```
Comment (marissadarnell) [inline frontend/src/components/pages/eventCard.tsx]: here as well

File: frontend/src/components/navigation/navBar.tsx [hunk 4]
```diff
@@ -12,46 +12,49 @@ import {
...
-      onSetOpen={setNavOpen}
-      styles={{
-        sidebar: { background: '#8ea974', width: '250px' },
-      }}
-    >
-      <Button onClick={() => setNavOpen(true)}>
-        <BarIcon />
-      </Button>
-    </Sidebar>
+    <StyledWrapper>
```
Comment (marissadarnell) [inline frontend/src/components/navigation/navBar.tsx]: can probably revert this

**Review by marissadarnell — CHANGES_REQUESTED** [review]
Hello hello! This overall looks good and seems to work just fine, nice job. I'm going to push the navbar changes onto here momentarily for you to work with. See my Slack message for more details on that, and of course please message me with questions, concerns, etc.

- [ ] undo all the z-index changes
- [ ] Fix navbar blocking screen for any other pages (ex. volunteer-log) (if you inspect in the browser you can observe if it is blocking other components)

**Comment by bbmcmann** [comment]
Reviewed the other pages, the navbar works on all the other pages it is needed on. The pages that currently have user interaction implemented on them work with he navbar attached. Thanks for figuring out that bug fix!

**Pull request merged**